docs/pages/project/contributing/contributing-cli.md

```diff
@@ -128,6 +128,25 @@ When designing for the command line interface, ask and consider the following qu
 
 You will need to provide a short and long description of the command for the help pages and also for the Meshery Documentation.
 
+## Refactoring Mesheryctl Code
+
+The mesheryctl code needs to be refactored for the ease of management, to improve the readbility of the code, for making it easier for using the utility packages. 
```

Comment:
This statement doesn't need to be made as this will have already been done by the time the contributor reads this.